src/packages/documents/documents/modals/rollback/rollback-modal.token.ts

```diff
@@ -0,0 +1,15 @@
+import { UMB_ROLLBACK_MODAL_ALIAS } from '../manifests.js';
+import { UmbModalToken } from '@umbraco-cms/backoffice/modal';
+
+export interface UmbRollbackModalData {}
+export interface UmbRollbackModalValue {}
+
+export const UMB_ROLLBACK_MODAL = new UmbModalToken<UmbRollbackModalData, UmbRollbackModalValue>(
+	UMB_ROLLBACK_MODAL_ALIAS,
+	{
+		modal: {
+			type: 'sidebar',
+			size: 'full',
```

Comment:
Does the `size` property actually do anything, and if so, is it supposed to be "full"?

Reply:
Size controls the width of the sidebar. Full means that it takes up the entire screen (except for a small gap on the left). Same as the old backoffice.